### ISSUE TITLE
No settings saved on close

### ISSUE THREAD

**Issue by MageSlayer** [uTox/uTox#733]
Tested under Windows x64, uTox v0.12.2 (https://build.tox.chat/view/uTox/job/uTox_build_windows_x86-64_release/lastSuccessfulBuild/artifact/utox_windows_x86-64.zip)
All settings are defaults after restart, contact list is clean and new id generated.

**Comment by cebe** [comment]
I guess this is the same as https://github.com/uTox/uTox/issues/732 then and somehow your uTox is not able to write to the disk at all?

**Comment by MageSlayer** [comment]
Might be the same issue, no idea.
Plenty of free space, run under Administrator account...

**Comment by robinlinden** [comment]
If you run uTox from the cmdline like so: `utox.exe -p`, does it save the Tox profile?
`-p` tells uTox to run in portable mode, so it should create a folder for the Tox save and all that in the same folder `utox.exe` is in.

**Comment by MageSlayer** [comment]
I get the following.
If utox is run from default installation location (C:\Users\Администратор\AppData\Roaming\uTox\uTox.exe), only Tox folder is created near utox.exe.

However, if run from C:\Temp\_tox\uTox.exe, files tox_save.tox and utox_save are also saved in Tox subfolder.

I am wondering, if it's related to russian symbols inside run/save path?

**Comment by robinlinden** [comment]
Yeah, you've likely solved it. It's probably because of the utf8 characters in the save path. We've fixed a similar issue in file transfers before, but I guess we missed this bug. Should be fixed shortly. :)

**Comment by redmanmale** [comment]
@MageSlayer 
One more reason not to use cyrillic user names.
There's pretty much apps (besides uTox) that still could not handle those correctly.

**Comment by MageSlayer** [comment]
@redmanmale 
Russian Windows has Administrator translated and there is nothing can be done about it from user POV.
And yes, it leads to never-ending issues.

**Comment by GrayHatter** [comment]
> There are pretty much apps (besides uTox) that still could not handle those correctly.

Glad to hear we're not the only ones. At the very least, we already know we can fix it. (The code already exits for incoming auto_accepted file transfers.

**Comment by GrayHatter** [comment]
@robinlinden did we fix this yet?

**Comment by robinlinden** [comment]
No, this isn't fixed yet.

**Comment by GrayHatter** [comment]
@uTox/devs we're now up to 5 duplicates on this bug...

**Comment by username1565** [comment]
Look at this: https://github.com/username1565/uTox_winXP/issues/6
